service/buildSrc/src/main/kotlin/TrevakaServiceVersion.kt

```diff
@@ -0,0 +1,4 @@
+object TrevakaServiceVersion {
+    const val kotlin = "1.4.21"
```

Comment:
Nämä oli ilmeisesti pakko laittaa Tampere-spesifisesti?

Reply:
This showcases a way of controlling plugin versions for the `trevaka-service` side of things with `buildSrc`-based centralized definitions. Unlike a multi-project, composite project structure does not seem to directly share `buildSrc` definitions across projects, so each project has their own separate definitions. I had some issues with IDE referencing the wrong `Version` objects in the evaka project structure, so I named the object more uniquely to prevent this on the trevaka side.